src/registry/selectboxes/edit.tsx

```diff
@@ -27,9 +27,64 @@ import {LABELS} from '@/components/builder/messages';
 import {SelectBoxes, TabList, TabPanel, Tabs} from '@/components/formio';
 import {useErrorChecker} from '@/utils/errors';
 
+import {NumberField} from '../../components/formio';
 import {EditFormDefinition} from '../types';
 import {checkIsManualOptions} from './helpers';
 
+const MinSelectedCheckboxes: React.FC = () => {
+  const intl = useIntl();
+  const tooltip = intl.formatMessage({
+    description: "Tooltip for 'validate.minSelectedCount' builder field",
+    defaultMessage: 'The minimum number of selected checkboxes.',
```

Comment:
```suggestion
    defaultMessage: 'If specified, the user must check at least this many options.',
```

and a similar message for the max selected count of course.